src/services/emailServices.js

```diff
@@ -1,20 +1,20 @@
-import { EMAIL_ADDR, EMAIL_PASSWORD } from '../config/constants';
+import { EMAIL_ADDR, EMAIL_PASSWORD, SENDER_ADDR } from '../config/constants';
```

Comment:
Remove the `EMAIL_ADDR` constant as it's no longer used

Reply:
The `EMAIL_ADDR` constant is still used.